hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/OperatingSystemMetricSet.java

```diff
@@ -53,22 +64,22 @@ public static void register(MetricsRegistry metricsRegistry) {
 
         OperatingSystemMXBean mxBean = ManagementFactory.getOperatingSystemMXBean();
 
-        registerMethod(metricsRegistry, mxBean, "getCommittedVirtualMemorySize", "os.committedVirtualMemorySize");
-        registerMethod(metricsRegistry, mxBean, "getFreePhysicalMemorySize", "os.freePhysicalMemorySize");
-        registerMethod(metricsRegistry, mxBean, "getFreeSwapSpaceSize", "os.freeSwapSpaceSize");
-        registerMethod(metricsRegistry, mxBean, "getProcessCpuTime", "os.processCpuTime");
-        registerMethod(metricsRegistry, mxBean, "getTotalPhysicalMemorySize", "os.totalPhysicalMemorySize");
-        registerMethod(metricsRegistry, mxBean, "getTotalSwapSpaceSize", "os.totalSwapSpaceSize");
-        registerMethod(metricsRegistry, mxBean, "getMaxFileDescriptorCount", "os.maxFileDescriptorCount");
-        registerMethod(metricsRegistry, mxBean, "getOpenFileDescriptorCount", "os.openFileDescriptorCount");
+        registerMethod(metricsRegistry, mxBean, "getCommittedVirtualMemorySize", OS_FULL_METRIC_COMMITTED_VIRTUAL_MEMORY_SIZE);
```

Comment:
Is it possible to specify `BYTES` unit here and for other memory related probes in this class?

Reply:
I tried this, but the OS metrics tests started failing so excluded it for now. Will take a look separately.

Reply:
Sure thing. I think that can be done in a separate PR.